pages/docs/how-to-guides/events.mdx

```diff
@@ -24,6 +24,10 @@ _If you need help on how to edit this file, please see the [Editing guide](/docs
 <ClipboardCopy>
 ```js
 {
+  "userStatus": "speaking",
```

Comment:
Maybe Add a separate pr for changes in docs? @eddiejaoude

Reply:
Ideally code changes and docs and tests should all be in the same PR if related